src/gitProviders/builtinGit.ts

```diff
@@ -30,17 +30,26 @@ export class BuiltinGitProvider implements IGit, vscode.Disposable {
 
 	private constructor(extension: vscode.Extension<GitExtension>) {
 		const gitExtension = extension.exports;
-		this._gitAPI = gitExtension.getAPI(1);
+
+		try {
+			this._gitAPI = gitExtension.getAPI(1);
+		} catch (e) {
+			// The git extension will throw if a git model cannot be found, i.e. if git is not installed.
+			vscode.window.showErrorMessage('Activating the Pull Requests and Issues extension failed. Please make sure you have git installed.');
+			throw e;
```

Comment:
Is this going to cause a error on web?

Reply:
Looks like probably not because of the if check in `createProvider` below.

Reply:
yeah, I think it should be handled there